packages/atomic/src/components.d.ts

```diff
@@ -330,6 +330,7 @@ export namespace Components {
     interface AtomicQuerySummary {
         /**
           * Whether to display the duration of the last query execution.
+          * @deprecated Use the `duration` part to display the duration instead.
```

Comment:
```suggestion
          * @deprecated Use the `duration` part.
```

Maybe my suggestion is a bit radical, but I do believe it would be enough to have just these 4 words :)